### DISCUSSION TITLE
Road to the v1.0 release

### DISCUSSION THREAD

**Discussion by mozman** [mozman/ezdxf#704] [Announcements]
The upcoming release  of v0.18  is the last version with new features before the release of v1.0. There will be one or two bugfix-releases for v0.18.x which may include minor API refactorings.

The following release cycle for v1.0 will start with some beta versions (b0, b1, ...) including **removal of deprecated APIs,** general clean-up work, refactoring and extending of documentation, examples and tutorials and further bugfixes, but no new features or API changes and will end with some release candidates (rc0, rc1, ...) before the final release of v1.0.

**Comment by chibai** [comment]
Finally! We get official v1.0!!!!
